core/src/main/java/com/netflix/conductor/service/MetadataService.java

```diff
@@ -233,7 +242,8 @@ public void removeEventHandlerStatus(String name) {
         return metadataDAO.getEventHandlersForEvent(event, activeOnly);
     }
```

Comment:
You might want to add validation for event.

Reply:
good catch, there are more methods in dao that could use extra validation and tests, will clean up